functions-and-operators/tidb-functions.md

```diff
@@ -230,3 +231,69 @@ See also:
 
 - [`Statement Summary Tables`](/statement-summary-tables.md)
 - [`INFORMATION_SCHEMA.TIDB_TRX`](/information-schema/information-schema-tidb-trx.md)
+
+### TIDB_SHARD
+
+The `TIDB_SHARD` function can be used to create a SHARD INDEX to scatter the index hotspot. A SHARD INDEX is an expression index with a `TIDB_SHARD`function as the prefix.
+
+#### SHARD INDEX
+
+- Creation:
+
+    When there is a hotspot caused by monotonically increasing or decreasing data on the index field `a` in the unique secondary index `uk((tidb_shard(a)), a))`, the index's prefix `tidb_shard(a)` can scatter the hotspot to improve the cluster's scalability.
+
+- Scenario:
+
+    - When there is a write hotspot caused by monotonically increasing or decreasing keys on the unique secondary index, and the index contains integer type fields.
+    - The SQL statement executes an equality query based on all fields of the secondary index, either as a separate `SELECT` or as an internal query generated by `UPDATE`, `DELETE` and so on. The equality query includes `a = 1` or `a IN (1, 2, ......)` these two ways.
+
+- Limitations:
+
+    - SHARD INDEX cannot be used in inequality queries.
```

Comment:
```suggestion
    - SHARD INDEX cannot be used in inequality or `IN` queries.
```

Reply:
Remove `IN` query in https://github.com/pingcap/docs-cn/pull/9632